db/re/item_db.conf

```diff
@@ -5855,9 +5855,11 @@ item_db: (
 	View: 3
 	Script: <"
 		bonus bFlee2,10;
-		if(JobLevel>=70) autobonus "{ bonus bBaseAtk,50; }",10,10000,BF_WEAPON,"{ specialeffect2 EF_POTION_BERSERK; }";
+		if(JobLevel=70)||(JobLevel>=50)&&(Class==Job_Rune_Knight||(JobLevel>=50)&&Class==Job_Rune_Knight_T) {
```

Comment:
On a side note, `JobLevel=70` is in any case incorrect (in case, it would be `==`)

The statement is also incorrectly parenthesized. The version suggested by @EPuncker seems correct.